README.md

```diff
@@ -154,14 +155,41 @@ This section explains how to configure Grafana alerts to notify you through emai
    Grafana begins to use the contact point to deliver alerts.
 
 <a name="updating-configuration"></a>
-## Updating Qumulo Monitoring Dashboard Configuration
-This section explains updating the configuration of the Qumulo Monitoring Dashboard.
+## Updating Your Qumulo Monitoring Dashboard Configuration
+This section explains how to update the configuration of the Qumulo Monitoring Dashboard for your system.
 
 <a name="updating-prometheus-configuration"></a>
-### Updating Prometheus Configuration
+### Updating Your Prometheus Configuration
+This section explains how to update the Prometheus configuration for your system.
+
 While Prometheus runs, it doesn't apply configuration changes automatically. To reload the configuration, you must do one of the following:
 
-* To stop and restart the container that Prometheus runs in, on your Docker host, run the following commands.
+* To stop and restart the container in which Prometheus runs on your Docker host, run the following commands.
+
+  **Note:** The `-d` flag runs the container in the background.
+
+  ```bash
+  docker-compose down
+  docker-compose up -d
+  ```
+
+* To make an HTTP `POST` call, use the `curl` command. For example:
+
+  ```bash
+  curl -X POST http://admin:admin@203.0.113.1:9090/-/reload
+  ```
+
+<a name="updating-grafana-configuration"></a>
+### Updating Your Grafana Configuration
+This section explains how to update the Grafana configuration for your system. To update the built-in Grafana alerts, you must modify their configuration files. To create new alerts, use the Grafana web UI.
+
+**Important:** Because the functionality of certain vendors' disks degrades before reaching 0% endurance, by default, the `low_disk_endurance` alert notifies when 20% endurance remains.
```

Comment:
This line seems odd without hinting at some action they can take. I think we should include some instruction to check with their hardware vendor about endurance thresholds, or some other indication that they may want to modify this value.

Also, `disk_endurance_low` is the file name, and the title in the UI is "Disk endurance low".

Reply:
All fixed. Merging.